### PR TITLE
fix: add core to hasura-auth-js dependencies

### PR DESCRIPTION
**Pull request by plmercereau**

### CODE REVIEW

**Comment by szilarddoro** [comment]
Why is this necessary? We are getting rid of the core package very soon, aren't we?

**Comment by elitan** [comment]
I don't know. It's hard to follow what's going on here.

I thought the core package was removed already, but now I see that that [PR](https://github.com/nhost/nhost/pull/1246) was merged into [`refactor/merge-core`](https://github.com/nhost/nhost/tree/refactor/merge-core) and the branch was removed. And there is an unanswered [question](https://github.com/nhost/nhost/pull/1246#pullrequestreview-1203745394) in the PR, but the PR was merged anyways.

Maybe someone with more visibility and knowledge of what's happening here can share their suggestion on what to do?

**Comment by szilarddoro** [comment]
> I don't know. It's hard to follow what's going on here.
> 
> 
> 
> I thought the core package was removed already, but now I see that that [PR](https://github.com/nhost/nhost/pull/1246) was merged into [`refactor/merge-core`](https://github.com/nhost/nhost/tree/refactor/merge-core) and the branch was removed. And there is an unanswered [question](https://github.com/nhost/nhost/pull/1246#pullrequestreview-1203745394) in the PR, but the PR was merged anyways.
> 
> 
> 
> Maybe someone with more visibility and knowledge of what's happening here can share their suggestion on what to do?

I think we shouldn't tweak with the packages until the PR for the major release is not merged. We should release those changes ASAP.

**Comment by plmercereau** [comment]
the `@nhost/core` package is being removed
